### PR TITLE
Update allowselfservicepurchase-powershell.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the document about requirements.

Tested in windows 11 without an admin account and it works.

Global reader connects to the module but does not work when running Update-MSCommerceProductPolicy

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7243

File: microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md [hunk 1]
```diff
@@ -37,10 +37,10 @@ You can use the **MSCommerce** PowerShell module to:
 
 To use the **MSCommerce** PowerShell module, you need:
 
-- A Windows 10 device
+- A Windows 10 or later Operating system.
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md]:
```suggestion
- A Windows 10 or later operating system.
```

File: microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md [hunk 2]
```diff
@@ -37,10 +37,10 @@ You can use the **MSCommerce** PowerShell module to:
...
 - PowerShell 5 or below. Currently, PowerShell 6.x/7.x isn't supported with this module.
-- Administrator permission for the device
-- Global or Billing Admin role for your tenant
+- Global or Billing Admin role for your tenant to to change the Commerce Product Policies.
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md]:
```suggestion
- Global or Billing admin role for your tenant to change the MSCommerce Product Policies.
```

File: microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md [hunk 3]
```diff
@@ -37,10 +37,10 @@ You can use the **MSCommerce** PowerShell module to:
...
+- Global reader to only read Commerce Product Policies.
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md]:
```suggestion
- Global reader to only read MSCommerce Product Policies.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thanks

File: microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md [hunk 4]
```diff
@@ -37,10 +37,10 @@ You can use the **MSCommerce** PowerShell module to:
 
 To use the **MSCommerce** PowerShell module, you need:
 
-- A Windows 10 device
+- A Windows 10 or later operating system.
 - PowerShell 5 or below. Currently, PowerShell 6.x/7.x isn't supported with this module.
-- Administrator permission for the device
-- Global or Billing Admin role for your tenant
+- Global or Billing admin role for your tenant to change the MSCommerce Product Policies.
```
Comment (cmcatee-MSFT) [inline microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md]: Change to:
The Global or Billing admin role for your tenant to change the **MSCommerce** product policies.

File: microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md [hunk 5]
```diff
@@ -37,10 +37,10 @@ You can use the **MSCommerce** PowerShell module to:
...
+- Global reader to only read MSCommerce Product Policies.
```
Comment (cmcatee-MSFT) [inline microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md]: Change to:
The Global reader role for your tenant to see a read-only list of **MSCommerce** product policies.

**Review by cmcatee-MSFT — CHANGES_REQUESTED** [review]
Please make the changes as indicated.

**Comment by msbemba** [comment]
@cmcatee-MSFT  Made changes per your request

File: microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md [hunk 6]
```diff
@@ -39,10 +39,10 @@ You can use the **MSCommerce** PowerShell module to:
 
 To use the **MSCommerce** PowerShell module, you need:
 
-- A Windows 10 device
+- A Windows 10 or later operating system.
 - PowerShell 5 or below. Currently, PowerShell 6.x/7.x isn't supported with this module.
-- Administrator permission for the device
-- Global or Billing Admin role for your tenant
+- Global or Billing admin role for your tenant to change the MSCommerce Product Policies.
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md:44]:
```suggestion
- The Global reader role for your tenant to see a read-only list of **MSCommerce** product policies.
```

Comment (JohanFreelancer9) [inline microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md]:
```suggestion
- The Global or Billing admin role for your tenant to change the **MSCommerce** product policies.
```

File: microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md [hunk 7]
```diff
@@ -39,10 +39,10 @@ You can use the **MSCommerce** PowerShell module to:
...
+- Global reader role for your tenant to see a read-only list of MSCommerce product policies.
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md]:
```suggestion
- The Global reader role for your tenant to see a read-only list of **MSCommerce** product policies.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@msbemba, please change as indicated. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thanks

**Review by cmcatee-MSFT — APPROVED** [review]

**Pull request merged**